src/gui/Layout.cpp

```diff
@@ -293,25 +293,24 @@ auto Layout::getPaddingLeftOfPage(size_t pageIndex) const -> int {
     bool isPairedPages = this->mapper.isPairedPages();
     const Settings* settings = this->view->getControl()->getSettings();
 
-    auto const paddingBefore =
-            sumIf(XOURNAL_PADDING, settings->getAddVerticalSpaceAmount(), settings->getAddHorizontalSpace());
+    auto paddingBefore =
+            sumIf(XOURNAL_PADDING, settings->getAddHorizontalSpaceAmount(), settings->getAddHorizontalSpace());
 
     auto const pageXLocation = as_signed(this->mapper.at(pageIndex).first);
 
     // No page pairing or we haven't rendered enough pages in the row for
     // page pairing to have an effect,
-    if (!isPairedPages || pageXLocation == 0) {
-        return strict_cast<int>(pageXLocation * XOURNAL_PADDING_BETWEEN + as_signed(paddingBefore));
+    if (!isPairedPages) {
+        return strict_cast<int>(pageXLocation * XOURNAL_PADDING_BETWEEN + XOURNAL_PADDING_BETWEEN / 2 +
+                                as_signed(paddingBefore));
     } else {
-        // We have a greater separation between pairs of pages. Handle this here,
-        //  Note that pageXLocation - 1 >= 0 because we take the if branch above when
-        // pageXLocation == 0.
-        auto paddingBetweenPairs = as_signed(pageXLocation - 1) / 2 * XOURNAL_PADDING_BETWEEN;
-
-        // The two pages within each pair have a smaller separation, XOURNAL_ROOM_FOR_SHADOW.
-        auto shadowRoomInsidePairs = pageXLocation / 2 * XOURNAL_ROOM_FOR_SHADOW;
-
-        return strict_cast<int>(paddingBetweenPairs + shadowRoomInsidePairs + paddingBefore);
+        auto columnPadding =
+                XOURNAL_PADDING_BETWEEN + strict_cast<int>(pageXLocation / 2) * (XOURNAL_PADDING_BETWEEN * 2);
+        if (pageXLocation % 2 == 0) {
+            return strict_cast<int>(columnPadding - XOURNAL_ROOM_FOR_SHADOW + paddingBefore);
+        } else {
+            return strict_cast<int>(columnPadding + XOURNAL_ROOM_FOR_SHADOW + paddingBefore);
+        }
```

Comment:
`columnPadding` should be equal to `XOURNAL_PADDING_BETWEEN`

Reply:
My version is correct AFAICT. See the layout code, to understand how this actually plays out.